src/Components/About.js

```diff
@@ -0,0 +1,307 @@
+import React, { useEffect } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import aos from "aos";
+import "aos/dist/aos.css";
+import { useMediaQuery } from "react-responsive";
+
+//import pictures nightmare
+import whatsapp from "../assets/logos/whatsapp.svg";
+import calender from "../assets/logos//calendar.svg";
+import email from "../assets/logos//envelope.svg";
+import telegram from "../assets/logos//telegram.svg";
+import location from "../assets/logos//location.svg";
+import linkedIn from "../assets/logos//linkedin.svg";
+import github from "../assets/logos//github.svg";
+import facebook from "../assets/logos//facebook.svg";
+import leetcode from "../assets/logos//leetcode.svg";
+
+function About() {
+  const isSmallScreen = useMediaQuery({ query: "(max-width: 359px)" });
+  let output;
+  useEffect(() => {
+    aos.init({ duration: 2000 });
+  }, []);
+  if (isSmallScreen) {
+    output = <SmallScreen />;
+  } else {
+    output = <Normal />;
+  }
```

Comment:
I feel like writing two separate components for different screen sizes is way too much effort. I would rather use media queries in your CSS to decide how to render the different elements on different screen sizes. Especially since you only got 2 screen sizes here.

```
@media (min-width: 360px) {
  .paragraph-intro {
     fontSize: max(1.4vw, 20px);
   }
}
```

If you did not do that, because it felt too bothersome, I recommend looking into styled-components. Those are components, to which you can pass props for styling purposes.